package.json

```diff
@@ -137,6 +137,8 @@
         "@loadable/babel-plugin": "^5.13.2",
         "@loadable/webpack-plugin": "^5.14.2",
         "@storybook/addon-essentials": "^6.2.8",
+        "@storybook/builder-webpack5": "^6.2.9",
+        "@storybook/manager-webpack5": "^6.3.0-alpha.37",
```

Comment:
Do we need to use the alpha release here?

Reply:
I don't think we need the manager-webpack5 at all, so I've removed it since there isn't a non-alpha version from what I can see.